common/testflinger_common/enums.py

```diff
@@ -22,6 +22,7 @@
 
 class JobState(StrEnum):
     WAITING = "waiting"
+    UNPACK = "unpack"
```

Comment:
should we call this "unpack" or some variation of "attachments"? I think the latter sounds more awkward but also makes it more obvious what's happening in this phase.

Either way, we should make sure the documentation is updated to reflect this new phase. In particular, in docs/reference/test-phases.rst and perhaps others that may describe results schema